plugins/org.eclipse.papyrus.uml.diagram.sequence.runtime/src/org/eclipse/papyrus/uml/diagram/sequence/runtime/internal/edit/parts/MessageEditPart.java

```diff
@@ -29,7 +29,9 @@
 import org.eclipse.uml2.uml.Message;
 import org.eclipse.uml2.uml.UMLPackage;
 
-public class MessageEditPart extends ConnectionNodeEditPart {
+public class MessageEditPart extends ConnectionNodeEditPart implements ISequenceEditPart {
+
+	private static final String ARROW = "_arrow"; //$NON-NLS-1$
```

Comment:
The `DefaultLayoutHelper` needs to know about this *modifier* constant.  Perhaps the modifiers should be enumerated somewhere?